front-end/src/components/newAccountForm.tsx

```diff
@@ -118,16 +119,21 @@ export function NewAccountForm(props: newAcccountFormProps) {
       .then((res) => {
         //User Login endpoint connection
         axios
-        .post("/api/user/login", {
-        username: emailField,
-        password: passwordField,
-        })
-        .then((loginRes) => {
-          history.push("home");
-        })
-        .catch(() => {
-          setAlertReason("Account created successfully, but login failed.");
-        })
+          .post("/api/user/login", {
+            username: emailField,
+            password: passwordField,
+          })
+          .then((loginRes) => {
+            //Checking the status of the response
+            const loginJSON = JSON.stringify(loginRes.data);
```

Comment:
Workarounds to fool typescript are punishable by 40 years in front-end prison. 
axios.post<any> instead of axios.post on line 122 should make it so you don't have to do this.

Reply:
axios.post instead of axios.post?

Reply:
Nevermind, just needed to refresh.

Reply:
Pushed [a18786c](https://github.com/cs481-ekh/f22-gneiss/pull/172/commits/a18786c3b459cdf73198806cbc63687aaa283fd9) to fix typescript workaround.